graph/vertex/ValueHolder.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Common interface implemented by Attribute & Value vertices. Used in Predicates & expressions to access the value.

Reply:
That breaks the symmetry a bit much doesn't it?